src/checkpoint/checkpoint_txn.c

```diff
@@ -1494,8 +1494,19 @@ __checkpoint_db_internal(WT_SESSION_IMPL *session, const char *cfg[])
      * view of the data, make sure that all the logs are flushed to disk before the checkpoint is
      * complete.
      */
-    if (F_ISSET(&conn->log_mgr, WT_LOG_ENABLED))
+    if (F_ISSET(&conn->log_mgr, WT_LOG_ENABLED)) {
```

Comment:
Can we leverage the logging variable set at line 1195 instead of re-checking WT_LOG_ENABLED?